tasks/php.yml

```diff
@@ -15,3 +15,59 @@
     dest: "/etc/init/php5-fpm.conf"
     line: "reload signal USR2"
   when: "ansible_distribution_version is version('14.04', '==')"
+
```

Comment:
Nice!

Reply:
Thanks Steve, I'll ask now a CR to the support team :)